pkgs/applications/misc/sweethome3d/default.nix

```diff
@@ -3,7 +3,7 @@
 , fetchurl
 , makeWrapper
 , makeDesktopItem
-, jdk
+, jdk19
```

Comment:
I think both `jdk19` and `jdk20` will quickly be marked as insecure in Nixpkgs. Is it possible to use `jdk17` instead?

Reply:
that would work, too